be/src/http/action/adjust_log_level.cpp

```diff
@@ -38,8 +38,18 @@ std::tuple<std::string, int, int> handle_request(HttpRequest* req) {
     };
     const auto& module = parse_param("module");
     const auto& level = parse_param("level");
-    int new_level = std::stoi(level);
-    return std::make_tuple(module, google::SetVLOGLevel(module.c_str(), new_level), new_level);
+    try {
+        int new_level = std::stoi(level);
```

Comment:
I searched the code base and find there are a lot `std::stoi`.
So I suggest to extract a method, like `safe_stoi()` to handle this,
so we don't need to catch these exception everywhere.